src/Cryptol/TypeCheck/Solver/Numeric.hs

```diff
@@ -137,6 +140,17 @@ tryGeqThanK _ t (Nat k) =
   -- XXX: K1 ^^ n >= K2
 
 
+-- (x >= 2 && x^a >= x^b) => a >= b
```

Comment:
The convention I've been using is to name things that only work for constants `K`, so maybe change the `x` to `K`.

Reply:
Updated in [110a753](https://github.com/GaloisInc/cryptol/pull/1713/commits/110a753a94506829ac11402ccf4f0067bb18bebe)